ui/org.eclipse.pde.bnd.ui/README.MD

```diff
@@ -26,7 +26,7 @@ transform an (Eclipse) `IProject` into a (bndlib) `Project` (from were the Works
 ```
 @Component
 @AdapterTypes(adaptableClass = IProject.class, adapterNames = Project.class)
-public class BndPluginAdapter implements IAdapterFactory {
+public class PdeBndAdapter implements IAdapterFactory {
```

Comment:
As this is an example for such an adapter, should be really mention PDE here?

```suggestion
public class MyBndProjectAdapter implements IAdapterFactory {
```

Reply:
Ah ok, I thought it was supposed to be the current PdeBndAdapter.java but just with an initial name. I was searching for the class in the readme (BndPluginAdapter) and did not find something. So I wanted to use a name from something which really exists, so I can have a look